code/modules/urist/random.dm

```diff
@@ -5,6 +5,22 @@
 		return 0
 	return 1
 
+//vg color setup stuff
+
+var/default_colour_matrix = list(1,0,0,0,\
+								 0,1,0,0,\
+								 0,0,1,0,\
+								 0,0,0,1)
+
+var/global/list/NOIRMATRIX = list(0.66,0.33,0.33,0,\
+				 				  0.33,0.33,0.33,0,\
+								  0.33,0.33,0.33,0,\
+								  0.00,0.00,0.00,1,\
+								  0.00,0.00,0.00,0)
+
+
+var/global/list/bad_changing_colour_ckeys = list()
```

Comment:
Same as above, except this time it's `GLOBAL_LIST_EMPTY(bad_changing_colour_ckeys)`